src/app/pages/cars/cars/cars-list-table-data-source.ts

```diff
@@ -226,19 +235,24 @@ export class CarsListTableDataSource extends TableDataSource<Car> {
       case CarButtonAction.CREATE_CAR:
         if (actionDef.action) {
           (actionDef as TableCreateCarActionDef).action(CarDialogComponent,
-            this.dialog, this.refreshData.bind(this));
+            this.dialog, { authorizations: this.authorizationActions }, this.refreshData.bind(this));
         }
         break;
     }
   }
 
   public buildTableFiltersDef(): TableFilterDef[] {
-    const tableFilterDef: TableFilterDef[] = [];
-    if (this.authorizationService.canListUsers()) {
-      tableFilterDef.push(new UserTableFilter().getFilterDef());
-    }
-    tableFilterDef.push(new CarMakerTableFilter().getFilterDef());
-    return tableFilterDef;
+    this.usersFilter = new UserTableFilter().getFilterDef();
+    this.carMakerFilter = new CarMakerTableFilter().getFilterDef();
+    // Filter visibility will be defined by auth
+    this.usersFilter.visible = false;
```

Comment:
Visible should be set to false by default. No need to set it explicitly.

Reply:
Ok !